### ISSUE TITLE
Unable to use violation_threshold to set expected violations

### ISSUE THREAD

**Issue by davotoula** [matemagyari/principle#5]
Great to see development is back on again!

I tried converting to the new yaml configuration format but I am unable to set expected violations using violation_threshold for layering and third_party_restrictions.

(I am able to set cyclic_dependencies_threshold and acd_threshold).

See error below:

```
violation_threshold: 5
...
[WARNING] 
==============================================================

Third party violations (5 of allowed 0 )    
==============================================================
...
```

Is is still work in progress or user error or bug?

fanatical user,

David

**Comment by matemagyari** [comment]
hey David, nice to see you here, certainly my most devoted user :) I will look into it and let you know

**Comment by matemagyari** [comment]
The issue is fixed! New version is 0.36

**Comment by davotoula** [comment]
Boom!

Both bad on the last day of 2016.

Happy New year to you and your family. Happy 2017

On 31 Dec 2016 16:56, "Mate Magyari" <notifications@github.com> wrote:

The issue is fixed! New version is 0.36

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/matemagyari/principle/issues/5#issuecomment-269870862>,
or mute the thread

**Comment by matemagyari** [comment]
Happy New Year, David!
